### ISSUE TITLE
Transitions funktionieren bei Firefox nicht

### ISSUE THREAD

**Issue by alexanderreiprich** [JirkaDellOro/FUDGE_Story#34]
Wie der Titel schon sagt, werden die Transitions bei Firefox nicht richtig angezeigt. So weit ich es ausprobiert habe, gibt es auch keinen Workaround, sodass die einzige Lösung ein Wechsel zu Chrome o.Ä. ist.

Dieses Problem würde ich nicht als unfassbar wichtig bezeichnen, da Firefox vergleichsweise mit Chrome wenige Nutzer hat, aber es ist schon nennenswert, damit andere Leute nicht stundenlang am gleichen Problem verzweifeln wie ich :]

**Comment by JirkaDellOro** [comment]
Gibt es eine Fehlermeldung in der Konsole? Poste bitte mal einen Lunk, so dass ich mit meinem Firefox testen kann.

**Comment by alexanderreiprich** [comment]
Eine Fehlermeldung gibt es in der Konsole nicht. Hier ist der Link zu meiner Visual Novel: https://alexanderreiprich.github.io/VisualNovel/

Bevor der erste Hintergrund angezeigt wird, soll eine Transition stattfinden. In Chrome funktioniert das fehlerfrei, in Firefox wird sie gar nicht angezeigt.

**Comment by JirkaDellOro** [comment]
@Rina14 ich weiß nicht mehr... lief das mal mit FireFox? Oder hatten wir FireFox an der Stelle als ungeeignet identifiziert?

**Comment by Rina14** [comment]
Nein, in Firefox laufen die Transitions leider nicht... Wir haben uns nur auf Chrome konzentriert.
